api.ts

```diff
@@ -1920,7 +2011,7 @@ export function makeMetadataFormula<ContextT extends ExecutionContext>(
     ],
     examples: [],
     connectionRequirement: options?.connectionRequirement || ConnectionRequirement.Optional,
-  });
+  }) as MetadataFormula<ContextT, ReturnT>;
```

Comment:
This was something that I resorted to due to TS complaining and not being able to resolve it another way. Left a comment above

Reply:
All makes sense, can you just add a comment above this line to remove the cast when `makeObjectFormula()` is removed?